### DISCUSSION TITLE
Is there a way to stop from getting into a Gait Mode? (ie Sprinting)

### DISCUSSION THREAD

**Discussion by wulfynArgox** [Sixze/ALS-Refactored#572] [Q&A]
I want to add an stamina system and I made the basics with float values and booleans. Now I want the ability to go into gait Sprinting when "CanSprint" boolen is false. However no matter what I do I can't do it. I've tried to look into the .cpp files but I didn't understand much. Where should I look or what should I do?

**Comment by xkamronx** [comment]
You can fake it by using the event OnGaitChanged and checking if you're current Gait is Sprinting, then if you don't meet your required conditions, use Set Desired Gait to go back to not sprinting.

It has a slight jitter but it works.
